### PR TITLE
Log out the CMakeOutputs.txt

### PR DESCRIPTION
**Pull request by Huy-Ngo**

### CODE REVIEW

**Comment by McSinyx** [comment]
This is weird:

    alure2_include_dirs=C:/projects/openal-soft-1.20.1-bin/include/AL;C:/Program Files (x86)/alure/include;C:/Program Files (x86)/alure/include/AL
    alure2_extra_objects=C:/projects/openal-soft-1.20.1-bin/libs/Win64/OpenAL32.lib;libalure2-NOTFOUND

**Comment by Huy-Ngo** [comment]
Good news: it can detect the dll now

```
alure2_include_dirs=C:/projects/openal-soft-1.20.1-bin/include/AL;C:/Program Files (x86)/alure/include;C:/Program Files (x86)/alure/include/AL
alure2_extra_objects=C:/projects/openal-soft-1.20.1-bin/libs/Win64/OpenAL32.lib;C:/Program Files (x86)/alure/bin/alure2.dll
```

Bad news: still library errors:

```
  C:/Program Files (x86)/alure/bin/alure2.dll : fatal error LNK1107: invalid or corrupt file: cannot read at 0x348
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\link.exe' failed with exit status 1107
  ----------------------------------------
  ERROR: Failed building wheel for palace
Failed to build palace
ERROR: Failed to build one or more wheels
```

**Comment by McSinyx** [comment]
See https://stackoverflow.com/q/8599539/5111569.  You can try to use [IMPORTED_IMPLIB](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_IMPLIB.html) but TBH I have no idea what's the proper way to do it on Windows.

**Comment by McSinyx** [comment]
Have you tried `IMPORTED_LOCATION_RELEASE` as suggested at https://github.com/kcat/alure/issues/43#issuecomment-629763405?

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,5 +6,5 @@ get_target_property(include_dirs Alure2::alure2 INTERFACE_INCLUDE_DIRECTORIES)
 message("alure2_include_dirs=${include_dirs}")
 
 get_target_property(link_libraries Alure2::alure2 INTERFACE_LINK_LIBRARIES)
-get_target_property(libalure2 Alure2::alure2 LOCATION)
+get_target_property(libalure2 Alure2::alure2 IMPORTED_IMLIB)
```
Comment (McSinyx) [inline CMakeLists.txt]: Oops this is a typo.

Comment (Huy-Ngo) [reply CMakeLists.txt]: IMPORTED_IMPLIB gives the same NOTFOUND thing

**Comment by Huy-Ngo** [comment]
What we tried:

- IMPORTED_IMPLIB: not found
- LOCATION: DLL on Windows, OK on GNU/Linux and macOS
- IMPORTED_LOCATION: not found
- IMPORTED_LOCATION_NOCONFIG: not found on Windows, OK on GNU/Linux and macOS
- IMPORTED_LOCATION_RELEASE: DLL on Windows, OK on macOS, GNU/Linux should work if installed as `Release`
- LOCATION_RELEASE: DLL on Windows, OK on macOS, GNU/Linux should work if installed as `Release`

**Comment by McSinyx** [comment]
Could you also note the result of each variable?  Edit: updated the list with corresponding outcomes.  Seemingly nothing really gives us the `.lib`s.

**Comment by Huy-Ngo** [comment]
I think I'm gonna try these:

- [IMPORTED_LIBNAME](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_LIBNAME.html)
- ~~[IMPORTED_LINK_DEPENDENT_LIBRARIES](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_LINK_DEPENDENT_LIBRARIES.html)~~ Not found
- [IMPORTED_LINK_INTERFACE_LIBRARIES](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_LINK_INTERFACE_LIBRARIES.html)
- ~~[IMPORTED_OBJECTS](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_OBJECTS.html)~~ Not found

**Comment by Huy-Ngo** [comment]
This got kinda stale, I guess I should close this

**Pull request closed without merging**